### PR TITLE
Add MyExpense

### PR DESCRIPTION
**Pull request by luturol**

### CODE REVIEW

File: src/myExpenses/Expenses.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class Expenses extends Component {
+  constructor(props){
+   super();
```
Comment (willmenn) [inline src/myExpenses/Expenses.js]: pass the props in the super.

File: src/myExpenses/MyExpense.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      whateverYouWant: 0
+    }
+  };
+
+  onSubmit(evt){
+    let body = Array.from(evt.target.elements)
+                       .filter(el => el.name)
+                       .reduce((a, b) => ({...a, [b.name]: b.value}), {});
+    evt.preventDefault();
+    this.setState({coin: body["coin"]});
```
Comment (willmenn) [inline src/myExpenses/MyExpense.js]: you can do only one `this.setState()`instead of 20.

File: src/myExpenses/YourSalary.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class YourSalary extends Component {
```
Comment (willmenn) [inline src/myExpenses/YourSalary.js]: The `Your` in the name of the class does not make much sense, try something like SalaryInput, or just Salary, I don't know the best name as well, but your is not good.

Comment (luturol) [reply src/myExpenses/YourSalary.js]: I also thought that it was strange

File: src/myExpenses/Expenses.js (hunk 1 above)
Comment (luturol) [reply src/myExpenses/Expenses.js]: removed unused constructor.

File: src/myExpenses/MyExpense.js (hunk 2 above)
Comment (luturol) [reply src/myExpenses/MyExpense.js]: done

File: src/myExpenses/YourSalary.js (hunk 3 above)
Comment (luturol) [reply src/myExpenses/YourSalary.js]: done

File: src/myExpenses/MyExpense.js [hunk 4]
```diff
@@ -22,14 +23,14 @@ class MyExpenses extends Component {
...
     evt.preventDefault();
-    this.setState({coin: body["coin"]});
-    this.setState({mySalary: body["salary"]});
-    this.setState({retirement: this.calculatePercent(5, body["salary"])});
-    this.setState({reserve: this.calculatePercent(5, body["salary"])});
-    this.setState({education: this.calculatePercent(10, body["salary"])});
-    this.setState({mediumTermDreams: this.calculatePercent(15, body["salary"])});
-    this.setState({essentialExpenses: this.calculatePercent(55, body["salary"])});
-    this.setState({whateverYouWant: this.calculatePercent(10, body["salary"])});
+    this.setState({coin: body["coin"],
```
Comment (willmenn) [inline src/myExpenses/MyExpense.js]: nice!

**Pull request merged**